superset/commands/report/base.py

```diff
@@ -76,3 +78,43 @@ def validate_chart_dashboard(
             self._properties["dashboard"] = dashboard
         elif not update:
             exceptions.append(ReportScheduleEitherChartOrDashboardError())
+
+    def validate_report_frequency(
+        self,
+        cron_schedule: str,
+        report_type: str,
+    ) -> None:
+        """
+        Validates if the report scheduled frequency doesn't exceed a limit
+        configured in `config.py`.
+
+        :param cron_schedule: The cron schedule configured.
+        :param report_type: The report type (Alert/Report).
+        """
+        config_key = (
+            "ALERT_MINIMUM_INTERVAL_MINUTES"
+            if report_type == ReportScheduleType.ALERT
+            else "REPORT_MINIMUM_INTERVAL_MINUTES"
+        )
+        minimum_interval = current_app.config.get(config_key)
```

Comment:
you may want to also perform some validation to make sure that this value is a number, or it will fail below at line 119.

Reply:
great catch! Just fixed that